.vscode/settings.template.json

```diff
@@ -0,0 +1,19 @@
+// Rename this file 'settings.json' or merge its
+// contents into your existing settings.
```

Comment:
Maybe worth a note in https://github.com/microsoft/TypeScript/blob/master/CONTRIBUTING.md ?

Reply:
Done